include/uv.h

```diff
@@ -1144,6 +1144,17 @@ UV_EXTERN int uv_os_getenv(const char* name, char* buffer, size_t* size);
 UV_EXTERN int uv_os_setenv(const char* name, const char* value);
 UV_EXTERN int uv_os_unsetenv(const char* name);
 
+#ifdef MAXHOSTNAMELEN
```

Comment:
Maybe the comment is unclear? Embedders should not be defining their own value. This is to avoid two `#includes` and some `#ifdef` logic every place that `MAXHOSTNAMELEN` is used (currently in libuv, Node's `os` module, and soon Node's `report` module.

If an embedder wants to use a different value, they would just pass that value as the `size` argument to `uv_os_gethostname()`.

Reply:
Ack.